composer.json

```diff
@@ -229,6 +229,9 @@
     "extra": {
         "enable-patching": true,
         "patches": {
+            "drupal/block_class": {
+                "Click JavaScript to close details causes modals to scroll unnecessarily https://www.drupal.org/project/block_class/issues/3497768": "https://git.drupalcode.org/project/block_class/-/merge_requests/60.patch"
```

Comment:
We have been avoiding git url patches because if the branch changes, it can cause problems. Download the patch into the repo and reference it via relative path.

Reply:
Changed to patch in repo.